Notification.astro

```diff
@@ -1,9 +1,13 @@
 ---
-const {
-  type = 'default',
-  role = 'none',
-  ariaLive = 'off'
-} = Astro.props
+interface Props {
+  // role="region" not supported as element takes no id
```

Comment:
@david-abell we wouldn't want to set a `role="region"` on this component 🤔 In this case the user will get a TypeScript error because it's not one of the options defined in the `interface Props` right?